### ISSUE TITLE
Error in base metadata file

### ISSUE THREAD

**Issue by atluxity** [cliffe/SecGen#111]
Hi, and thanks for a great project.
I just now checked out the latest master branch and found that I was unable to build any projects. I am able to do this with the previous commit in the master branch. This is the message I get:

atluxity@laptop:~/SecGen$ ruby secgen.rb --scenario scenarios/ctf/basic_narrative.xml build-project
<scroll>
<scroll>
1 system(s) specified
Reading available base modules...
 Reading base: modules/bases/debian_puppet_32_desktop
Error in base metadata file (/home/atluxity/SecGen/modules/bases/debian_puppet_32_desktop/secgen_metadata.xml):
    Element '{http://www.github/cliffe/SecGen/base}ovirt_template': This element is not expected. Expected is one of ( {http://www.github/cliffe/SecGen/base}software_license, {http://www.github/cliffe/SecGen/base}conflict, {http://www.github/cliffe/SecGen/base}requires ).
Failed to validate base metadata file (/home/atluxity/SecGen/modules/bases/debian_puppet_32_desktop/secgen_metadata.xml): against schema (/home/atluxity/SecGen/lib/schemas/base_metadata_schema.xsd)
exit

This is the same for any scenario I try.

**Comment by atluxity** [comment]
For those encountering the same issue and need a workaround:
git checkout a49793a4cb

This will checkout a previous git version of the project, and is just a temporary workaround until this issue is fixed.

**Comment by cliffe** [comment]
Thanks for raising this, @atluxity. 

I screwed up with git... The system (secgen_server) I was on had a 'diverged' git history from master. Seemed safe to push a merge that just resolved a conflicting commit, but there must have been more going on than I realised. 

@thomashaw, should this just be reverted with a soft head reset? It looks like the new merge commit has a bunch of stuff such as Hackerbot that I thought was already merged into master? Is this related to the fact we had a squashed commit from the labs branch at some point?. 

@atluxity we will sort this out tomorrow. We do try to keep master branch in a working state.
